src/backend/routers/auth.py

```diff
@@ -317,3 +317,74 @@ def log_and_redirect_err(error_message: str):
     response = RedirectResponse(redirect_uri)
 
     return response
+
+
+@router.delete("/tool/auth/{tool_id}")
+async def delete_tool_auth(
+    tool_id: str,
+    request: Request,
+    session: DBSessionDep,
+    ctx: Context = Depends(get_context),
+):
+    """
+    Endpoint to delete Tool Authentication.
+
+    If completed, the corresponding ToolAuth for the requesting user is removed from the DB.
+
+    Args:
+        tool_id (str): ToolID to be deleted for the user. (eg. google_drive) Should be one of the values listed in the ToolName string enum class.
+        request (Request): current Request object.
+        session (DBSessionDep): Database session.
+        ctx (Context): Context object.
+
+    Returns:
+        {"status": "success"} on successful deletion.
```

Comment:
To correspond to other DELETE endpoints, let's just return a `{}`. Usually we do this by adding a new pydantic schema with nothing in it. You can add a

```
class DeleteToolAuth(BaseModel):
    pass
```

pydantic schema and return that

Reply:
Ah, I didn't know about this, thank you!